### PR TITLE
Improve README

### PR DESCRIPTION
**Pull request by samoht**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,28 +31,27 @@ wrapped up in the tool.
 To work with Mirage, you'll need the following prerequisites installed:
 
 * a working [OCaml](http://ocaml.org) compiler (4.01.0 or higher).
-* the [OPAM](https://opam.ocaml.org) source package manager (1.2.0 or higher).
-* an x86\_64 or armel Linux host to compile Xen kernels, or FreeBSD, OpenBSD or MacOS X
-  for the userlevel version.
+* the [OPAM](https://opam.ocaml.org) source package manager (1.2.0 or
+  higher).
+* an x86\_64 or armel Linux host to compile Xen kernels, or FreeBSD,
```
Comment (hannesm) [inline README.md]: what is `armel`?

File: README.md [hunk 2]
```diff
@@ -72,41 +77,34 @@ To build for the Xen target, do:
...
-
-## Building Mirage Applications
+### Building Mirage Applications
 
 The command:
 
 ```
-mirage build
+make
 ```
```
Comment (hannesm) [inline README.md]: here there should be some sentence... `The command make will compile your unikernel` or sth?

Comment (samoht) [inline README.md]: I've added a sentence, thx

File: README.md (hunk 1 above)
Comment (samoht) [inline README.md]: I think this refers to https://wiki.debian.org/ArmEabiPort

Comment (samoht) [inline README.md]: Not sure who added that though and if is it still needed.

**Pull request merged**